RabbitMQ.Stream.Client/RoutingClient.cs

```diff
@@ -100,6 +99,25 @@ public static class RoutingHelper<T> where T : IRouting, new()
             return client;
         }
 
+        internal static async Task<EndPoint> GetEndPoint(Broker broker)
+        {
+            switch (Uri.CheckHostName(broker.Host))
+            {
+                case UriHostNameType.Basic:
+                case UriHostNameType.Dns:
+                    var hostEntry = await Dns.GetHostEntryAsync(broker.Host);
+                    var endPointNoLb = new IPEndPoint(hostEntry.AddressList.First(), (int)broker.Port);
```

Comment:
Is it allowed for a user to configure DNS to use the same host name for a cluster, and configure multiple IPs for that host name? If so, we should select a random IP returned here.

Reply:
We have `first()` in production for a while https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/blob/main/RabbitMQ.Stream.Client/RoutingClient.cs#L63

Should we change it?

Reply:
Let's not bother until someone requests the ability to configure a DNS host with multiple IP addresses